src/components/containers/SSIButtonsContainer/index.tsx

```diff
@@ -41,11 +42,11 @@ class SSIButtonsContainer extends PureComponent<Props, IState> {
   };
 
   render() {
-    const {primaryButton, secondaryButton} = this.props;
+    const {primaryButton, secondaryButton, style} = this.props;
     const {keyboardVisible} = this.state;
 
     return (
-      <ButtonContainer style={{marginBottom: keyboardVisible ? 18 : 36}}>
+        <ButtonContainer style={{marginBottom: keyboardVisible ? 18 : 36, ...(style && style)}}>
```

Comment:
style={{...style, marginBottom: keyboardVisible ? 18 : 36}}

you can just deconstruct in this case

Reply:
removed this styling